src/pages/DownloadMonitor.vue

```diff
@@ -19,18 +19,21 @@
                         <div class="border-at-bottom pad pad--sides">
                             <div class="card is-shadowless">
                                 <p><strong>{{ downloadObject.initialMods.join(", ") }}</strong></p>
-                                <p v-if="downloadObject.progress < 100">Downloading: {{ downloadObject.modName }}</p>
-                                <p>{{Math.min(Math.floor(downloadObject.progress), 100)}}% complete</p>
-                                <Progress v-if="!downloadObject.failed"
+                                <p v-if="downloadObject.failed">Downloading failed</p>
+                                <p v-else-if="downloadObject.progress < 100">Downloading: {{ downloadObject.modName }}</p>
```

Comment:
This doesn't update `modName` when dependencies are being downloaded but I think that is addressed in #1625